cmd/agent/app/reporter/grpc/builder_test.go

```diff
@@ -194,7 +195,7 @@ func TestProxyBuilder(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			proxy, err := NewCollectorProxy(test.grpcBuilder, nil, metrics.NullFactory, zap.NewNop())
+			proxy, err := NewCollectorProxy(test.grpcBuilder, nil, reporter.Duplicate, metrics.NullFactory, zap.NewNop())
```

Comment:
please change the signature to accept a single `CollectorProxyParams` struct, since this is a breaking change anyway